src/Eloquent/Builder.php

```diff
@@ -164,10 +164,10 @@ public function chunkById($count, callable $callback, $column = '_id', $alias =
     /**
      * @inheritdoc
      */
-    public function raw($expression = null)
+    public function raw($value = null)
```

Comment:
I assume this is related to https://github.com/GromNaN/laravel-mongodb/pull/26/files#r1300010923.